test/schemas_misc.jl

```diff
@@ -209,7 +209,7 @@ using Braket, Braket.IR, Test, JSON3, StructTypes
     end
 
     @testset "Device metadata" begin
-        @testset for T in (Braket.OqcMetadata, Braket.XanaduMetadata, Braket.RigettiMetadata)
+        @testset for T in (Braket.OqcMetadata, Braket.IqmMetadata, Braket.XanaduMetadata, Braket.RigettiMetadata)
```

Comment:
Should we drop `OqcMetadata` here?

Reply:
Maybe? `XanaduMetadata` is still here. I'd be happy to make a pass through and remove all of the retired devices.

Reply:
I guess I'm wondering if we're still testing these on the Python side

Reply:
Good point. No, we're not. The code for the retired devices is still there for backward compatibility, but the tests have been removed (example: https://github.com/amazon-braket/amazon-braket-sdk-python/pull/1014). I can make the same change here if it makes sense to you.